docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -254,9 +258,16 @@ for the application.
 - `alluxio.zookeeper.address=<ZOOKEEPER_ADDRESS>`
   - The ZooKeeper address must be specified when `alluxio.zookeeper.enabled` is enabled.
   - Multiple ZooKeeper addresses can be specified by delimiting with commas
+  
+If using embedded journal, users can configure the following parameter.
+
+- `alluxio.master.rpc.addresses=master_hostname_1:19998;master_hostname_2:19998;master_hostname_3:19998`
+
+If both parameters are set, the Zookeeper configuration always take precedence. 
```

Comment:
We might change this behavior to give an error if both embedded and Zookeeper configuration is set. Can we leave the behavior undefined by not specifying it here?